Bottom: cd61bfbc2c99b1f91308eee9fbeff14699c5ada4
Top:    03e0d857be2abeca4aaf50a1c372dc983b1332dc
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-03-05 17:44:49 -0800

Refresh of release

---

diff --git a/configure.in b/configure.in
index b72d0aa..35d79dd 100644
--- a/configure.in
+++ b/configure.in
@@ -5,7 +5,7 @@ AC_INIT(librdmacm, 1.0.17, linux-rdma@vger.kernel.org)
 AC_CONFIG_SRCDIR([src/cma.c])
 AC_CONFIG_AUX_DIR(config)
 AC_CONFIG_MACRO_DIR(config)
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 AM_INIT_AUTOMAKE(librdmacm, 1.0.17)
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
